base/models.py

```diff
@@ -495,10 +562,10 @@ def autorizzazione_richiedi_sede_riferimento(self, richiedente, incarico, invia_
             ufficio_soci = list(sede_riferimento.delegati_ufficio_soci())
             invia_notifiche = list(invia_notifiche) + ufficio_soci
 
-        self.autorizzazione_richiedi(richiedente, (incarico, sede_riferimento), invia_notifiche=invia_notifiche, **kwargs)
+        self.autorizzazione_richiedi(richiedente, (incarico, sede_riferimento), invia_notifiche=invia_notifiche, auto=auto, **kwargs)
         return True
 
-    def autorizzazione_richiedi(self, richiedente, destinatario, invia_notifiche=None, **kwargs):
+    def autorizzazione_richiedi(self, richiedente, destinatario, invia_notifiche=None, auto=None, scadenza_giorni=None, **kwargs):
```

Comment:
@AlfioEmanueleFresta il doppio parametro in realtà serve ad impostare il tipo di gestione automatica ( negazione o accettazione ), informazione che la scadenza sola non può contenere.